snippets/language-python.cson

```diff
@@ -88,10 +88,16 @@
     'body': '{${1:key}: ${2:value} for ${3:key}, ${4:value} in ${5:variable}}'
   'PDB set trace':
     'prefix': 'pdb'
-    'body': 'import pdb; pdb.set_trace()\n'
+    'body': 'import pdb; pdb.set_trace()'
   'iPDB set trace':
     'prefix': 'ipdb'
-    'body': 'import ipdb; ipdb.set_trace()\n'
+    'body': 'import ipdb; ipdb.set_trace()'
+  'rPDB set trace - ':
```

Comment:
Hold on on that merge, seems I committed mid-work. I'll commit the rest as soon as I get home.